### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by sinhprab**
VPC creation with required resources

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.terraform.lock.hcl
+.terraform/
+terraform.tfstate
+terraform.tfstate.backup
```
Comment (vladimirmukhin) [inline .gitignore:4]: Please add trailing newline

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # terraform-in-21-days
-my 1st commit !
+my 1st commit !brew update
```
Comment (vladimirmukhin) [inline README.md:2]: No need to update README.md anymore. In classes it was just done for the demo purposes

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  tags = {
+    Name = "private02"
+  }
+}
+
+resource "aws_route_table_association" "public01" {
+  subnet_id      = aws_subnet.public01.id
+  route_table_id = aws_route_table.public.id
+
+}
```
Comment (vladimirmukhin) [inline main.tf]:
```suggestion
  route_table_id = aws_route_table.public.id
}
```

File: main.tf [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+resource "aws_route_table_association" "public02" {
+  subnet_id      = aws_subnet.public02.id
+  route_table_id = aws_route_table.public.id
+
+}
```
Comment (vladimirmukhin) [inline main.tf]:
```suggestion
  route_table_id = aws_route_table.public.id
}
```

File: main.tf [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+resource "aws_route_table_association" "private01" {
+  subnet_id      = aws_subnet.private01.id
+  route_table_id = aws_route_table.private01.id
+
+}
```
Comment (vladimirmukhin) [inline main.tf]:
```suggestion
  route_table_id = aws_route_table.private01.id
}
```

File: main.tf [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+
+resource "aws_route_table_association" "private02" {
+  subnet_id      = aws_subnet.private02.id
+  route_table_id = aws_route_table.private02.id
+
+}
```
Comment (vladimirmukhin) [inline main.tf]:
```suggestion
  route_table_id = aws_route_table.private02.id
}
```

File: provider.tf [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+      source  = "hashicorp/aws"
+      version = "5.6.2"
+    }
+  }
+  required_version = ">= 1.2.0"
+}
+
+provider "aws" {
+  region = "eu-west-2"
+}
```
Comment (vladimirmukhin) [inline provider.tf]: Please add trailing newline

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

File: provider.tf (hunk 7 above)
Comment (sinhprab) [reply provider.tf]: Updated as asked

File: main.tf (hunk 6 above)
Comment (sinhprab) [reply main.tf]: Updated as asked

File: main.tf (hunk 5 above)
Comment (sinhprab) [reply main.tf]: Updated as asked

File: main.tf (hunk 4 above)
Comment (sinhprab) [reply main.tf]: Updated as asked

File: main.tf (hunk 3 above)
Comment (sinhprab) [reply main.tf]: Updated as asked

File: README.md (hunk 2 above)
Comment (sinhprab) [reply README.md:2]: Noted !

**Review by vladimirmukhin — APPROVED** [review]

**Pull request merged**